mask/eolearn/tests/test_cloud_mask.py

```diff
@@ -10,11 +10,34 @@
 """
 
 import numpy as np
+import pytest
 from numpy.testing import assert_array_equal
 from pytest import approx
 
 from eolearn.core import FeatureType
 from eolearn.mask import CloudMaskTask
+from eolearn.mask.cloud_mask import _get_window_indices
+
+
+@pytest.mark.parametrize(
+    "num_of_elements, middle_idx, window_size, expected_indices",
+    (
+        [100, 0, 10, (0, 10)],
+        [100, 1, 10, (0, 10)],
+        [100, 50, 10, (45, 55)],
+        [100, 99, 10, (90, 100)],
+        [100, 100, 10, (90, 100)],
+        [100, 0, 11, (0, 11)],
+        [100, 1, 11, (0, 11)],
+        [100, 50, 11, (45, 56)],
+        [100, 99, 11, (89, 100)],
+        [100, 100, 11, (89, 100)],
+    ),
+)
+def test_window_indices_function(num_of_elements, middle_idx, window_size, expected_indices):
+    indices = _get_window_indices(num_of_elements, middle_idx, window_size)
+    assert indices == expected_indices
+    assert len(range(*indices)) == window_size
```

Comment:
since these are meant to be indices, how about you construct a list/numpy array of the appropriate length and then apply these indices :thinking: This way tests are also an indicator of how we expect the result to be used. Literature often suggests that test should also serve as documentation (and we're not there yet)

Reply:
updated now